src/current/_includes/v23.1/known-limitations/cdc-execution-locality.md

```diff
@@ -0,0 +1 @@
+Changefeeds that use the [`execution_locality` option]({% link {{ page.version.version }}/changefeeds-in-multi-region-deployments.md %}#run-a-changefeed-job-by-locality) set to a [secondary region]({% link {{ page.version.version }}/multiregion-overview.md %}#secondary-regions) could create a plan that assigns most of the ranges to an [aggregator]({% link {{ page.version.version }}/how-does-an-enterprise-changefeed-work.md %}) on the coordinator node. This leads to an unbalanced plan and slow changefeed progress, particularly when the table is large and has many ranges. [Tracking GitHub Issue](https://github.com/cockroachdb/cockroach/issues/124822)
```

Comment:
I'm not sure if this works but could you include `src/current/_includes/v23.1/cdc/cdc-execution-locality.md` here so that you don't have to duplicate that text and then you'd only need to append the tracking issue?

Reply:
It's possible! I didn't do that mainly because 1) we're on a new kick of grouping known limitations in this directory under a separate file. 2) I get a bit scared of having trailing links after include file references in case they get lost. I could have included some logic of `if this page show github issue else...` but that tends to get messy. I'll give it another think though, and see if @Amruta-Ranade  (docs reviewer) has a better idea. Thanks so much for looking at the file structure/logic too Andy, really helpful!

Reply:
I agree with your reasoning, @kathancox!